aws-lambda-extension-fluent-bit/README.md

```diff
@@ -0,0 +1,87 @@
+# Fluent Bit Lambda Extensions in Go
+
+The provided code sample demonstrates how to get Fluent Bit set up for Lambda Extensions. 
+
+Note: 
+1. Supported Fluent Bit: v1.6
```

Comment:
This is not true. My code changes have not been merged yet, but once they are, it will mean Lambda is supported in Fluent Bit version 1.7.4 or greater

Reply:
Also, link to the official Fluent Bit docs somewhere in the beginning